Gemfile

```diff
@@ -27,15 +27,16 @@ gem 'ransack'
 gem 'friendly_id'
 gem 'babosa'
 gem 'bulma-rails'
+gem 'bulma-extensions-rails'
 gem 'webpacker'
 gem 'react_on_rails'
 gem 'omniauth'
 gem 'omniauth-facebook'
 gem 'annotate'
 gem 'ranked-model'
 gem 'meta-tags'
-gem 'jquery-rails'
 gem "paranoia", '~> 2.2'
+# gem 'flatpickr_rails'
```

Comment:
eh?

Reply:
its for next task "flatpickr" datepicker